upgrade/helpers/tasks.py

```diff
@@ -998,3 +998,45 @@ def upgrade_validation(upgrade_type=False):
     if upgrade_type:
         run('hammer ping', warn_only=True)
     run('katello-service status', warn_only=True)
+
+
+def update_scap_content():
+    """ The purpose of this function is to perform deletion of old scap-contents
+        and then uploading new scap-contents. It also deletes scap-policies and creates
+         new scap-policies with new scap-contents. """
+
+    def create_policy(scap_content, policy_name):
+        scap_content_profile_id = entities.ScapContents(
+            id=scap_content.id).read().scap_content_profiles[0]['id']
+        entities.CompliancePolicies(
+            name=policy_name,
+            scap_content_id=scap_content.id,
+            scap_content_profile_id=scap_content_profile_id,
+            deploy_by='puppet',
+            organization=[1],
+            location=[2],
```

Comment:
I think we should need to extract these information instead of presuming the default value.
what do you say ?

Reply:
Makes sence, I have updated the PR